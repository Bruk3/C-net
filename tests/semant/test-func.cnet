int main(int argc) {
    int a;
    a = 4;
    return 5*4;
}