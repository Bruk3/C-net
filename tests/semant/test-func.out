int main()
{
int a = (int : 0);
(int : a=(int : 4));
return (int : (int : 5) * (int : 4));
}

